locale/circuitpython.pot

```diff
@@ -1518,6 +1522,10 @@ msgstr ""
 msgid "UART Re-init error"
 msgstr ""
 
+#: ports/mimxrt10xx/common-hal/busio/UART.c
+msgid "UART peripheral is already in use"
```

Comment:
Please reuse error messages from other ports when it makes sense. That way there are less strings to translate. Here is what STM does: https://github.com/adafruit/circuitpython/blob/master/ports/stm/common-hal/busio/UART.c#L58